src/main/rx/jetiexbus.c

```diff
@@ -212,9 +222,13 @@ static void jetiExBusDataReceive(uint16_t c, void *data)
 
     // Done?
     if (jetiExBusFrameLength == jetiExBusFramePosition) {
```

Comment:
Here is cause of the overflow problem - jetiExBusFrame[EXBUS_HEADER_MSG_LEN] can be smaller than EXBUS_HEADER_LEN - this test will never trigger and buffer will overflow
```suggestion
    if (jetiExBusFramePosition >= jetiExBusFrameLength) {
```